<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1920</width>
    <height>1080</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>1920</width>
    <height>1080</height>
   </size>
  </property>
  <property name="cursor">
   <cursorShape>PointingHandCursor</cursorShape>
  </property>
  <property name="contextMenuPolicy">
   <enum>Qt::NoContextMenu</enum>
  </property>
  <property name="windowTitle">
   <string notr="true">MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <layout class="QFormLayout" name="formLayout">
    <item row="0" column="0">
     <spacer name="horizontalSpacer_4">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>10</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="0" column="1">
     <widget class="QStackedWidget" name="pages">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
        <horstretch>190</horstretch>
        <verstretch>110</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>1920</width>
        <height>1080</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>1920</width>
        <height>16777214</height>
       </size>
      </property>
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="styleSheet">
       <string notr="true">left-margin:0 px;</string>
      </property>
      <property name="lineWidth">
       <number>0</number>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="main">
       <property name="minimumSize">
        <size>
         <width>1920</width>
         <height>1080</height>
        </size>
       </property>
       <widget class="QLabel" name="companyName">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>30</y>
          <width>239</width>
          <height>77</height>
         </rect>
        </property>
        <property name="maximumSize">
         <size>
          <width>250</width>
          <height>120</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Ubuntu</family>
          <pointsize>51</pointsize>
         </font>
        </property>
        <property name="toolTipDuration">
         <number>-1</number>
        </property>
        <property name="lineWidth">
         <number>0</number>
        </property>
        <property name="text">
         <string>drinkfill</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
        <property name="indent">
         <number>0</number>
        </property>
       </widget>
       <widget class="QWidget" name="layoutWidget">
        <property name="geometry">
         <rect>
          <x>-9</x>
          <y>150</y>
          <width>1921</width>
          <height>122</height>
         </rect>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLabel" name="productToDispense">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="text">
            <string>TextLabel</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Preferred</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
       <widget class="QLabel" name="label">
        <property name="geometry">
         <rect>
          <x>1770</x>
          <y>1010</y>
          <width>121</width>
          <height>51</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>15</pointsize>
         </font>
        </property>
        <property name="text">
         <string>CONTACT US</string>
        </property>
       </widget>
       <widget class="QWidget" name="layoutWidget">
        <property name="geometry">
         <rect>
          <x>-2</x>
          <y>111</y>
          <width>1941</width>
          <height>871</height>
         </rect>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QToolButton" name="sparklingwater">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>550</width>
             <height>720</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>550</width>
             <height>720</height>
            </size>
           </property>
           <property name="font">
            <font>
             <pointsize>25</pointsize>
             <weight>50</weight>
             <bold>false</bold>
             <underline>false</underline>
             <stylestrategy>PreferAntialias</stylestrategy>
             <kerning>true</kerning>
            </font>
           </property>
           <property name="autoFillBackground">
            <bool>false</bool>
           </property>
           <property name="styleSheet">
            <string notr="true"/>
           </property>
           <property name="text">
            <string>WATER</string>
           </property>
           <property name="icon">
            <iconset>
             <normaloff>images/page0/carbonated-button.png</normaloff>images/page0/carbonated-button.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>550</width>
             <height>720</height>
            </size>
           </property>
           <property name="toolButtonStyle">
            <enum>Qt::ToolButtonIconOnly</enum>
           </property>
           <property name="autoRaise">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="kombucha">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>550</width>
             <height>720</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>550</width>
             <height>720</height>
            </size>
           </property>
           <property name="font">
            <font>
             <pointsize>25</pointsize>
            </font>
           </property>
           <property name="text">
            <string>KOMBUCHA</string>
           </property>
           <property name="icon">
            <iconset>
             <normaloff>images/page0/kombucha-button.png</normaloff>images/page0/kombucha-button.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>550</width>
             <height>720</height>
            </size>
           </property>
           <property name="toolButtonStyle">
            <enum>Qt::ToolButtonIconOnly</enum>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="coffee">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>550</width>
             <height>720</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>550</width>
             <height>720</height>
            </size>
           </property>
           <property name="font">
            <font>
             <pointsize>25</pointsize>
            </font>
           </property>
           <property name="autoFillBackground">
            <bool>false</bool>
           </property>
           <property name="styleSheet">
            <string notr="true"/>
           </property>
           <property name="text">
            <string>COLD COFFEE</string>
           </property>
           <property name="icon">
            <iconset>
             <normaloff>images/page0/Coldbrew-button.png</normaloff>images/page0/Coldbrew-button.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>550</width>
             <height>720</height>
            </size>
           </property>
           <property name="toolButtonStyle">
            <enum>Qt::ToolButtonIconOnly</enum>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
      <widget class="QWidget" name="selected">
       <property name="minimumSize">
        <size>
         <width>1900</width>
         <height>1080</height>
        </size>
       </property>
       <widget class="QPushButton" name="back">
        <property name="geometry">
         <rect>
          <x>12</x>
          <y>12</y>
          <width>80</width>
          <height>50</height>
         </rect>
        </property>
        <property name="minimumSize">
         <size>
          <width>50</width>
          <height>50</height>
         </size>
        </property>
        <property name="text">
         <string>back</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_2">
        <property name="geometry">
         <rect>
          <x>12</x>
          <y>796</y>
          <width>67</width>
          <height>17</height>
         </rect>
        </property>
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
       <widget class="QLabel" name="productImage">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>120</y>
          <width>520</width>
          <height>720</height>
         </rect>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>350</width>
          <height>712</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>520</width>
          <height>720</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>1</pointsize>
         </font>
        </property>
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;img src=&quot;:/images/page1/carbonated.png&quot;/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <property name="textFormat">
         <enum>Qt::RichText</enum>
        </property>
        <property name="scaledContents">
         <bool>true</bool>
        </property>
        <property name="textInteractionFlags">
         <set>Qt::NoTextInteraction</set>
        </property>
       </widget>
       <widget class="QGroupBox" name="buttonGroup_2">
        <property name="geometry">
         <rect>
          <x>740</x>
          <y>170</y>
          <width>901</width>
          <height>311</height>
         </rect>
        </property>
        <property name="autoFillBackground">
         <bool>true</bool>
        </property>
        <property name="title">
         <string/>
        </property>
        <widget class="QPushButton" name="pushButton_1">
         <property name="geometry">
          <rect>
           <x>50</x>
           <y>30</y>
           <width>250</width>
           <height>250</height>
          </rect>
         </property>
         <property name="minimumSize">
          <size>
           <width>250</width>
           <height>250</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>25</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton {
    color: #333;
    border: 0px solid #555;
    border-radius: 125px;
    border-style: outset;
    background: white;
    padding: 5px;
    }
</string>
         </property>
         <property name="text">
          <string>16oz
473ml</string>
         </property>
         <attribute name="buttonGroup">
          <string notr="true">buttonGroup</string>
         </attribute>
        </widget>
        <widget class="QPushButton" name="pushButton_2">
         <property name="geometry">
          <rect>
           <x>340</x>
           <y>30</y>
           <width>250</width>
           <height>250</height>
          </rect>
         </property>
         <property name="minimumSize">
          <size>
           <width>250</width>
           <height>250</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>25</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton {
    color: #333;
    border: 0px solid #555;
    border-radius: 125px;
    border-style: outset;
    background: white;
    padding: 5px;
    }
</string>
         </property>
         <property name="text">
          <string>24oz
709ml</string>
         </property>
         <attribute name="buttonGroup">
          <string notr="true">buttonGroup</string>
         </attribute>
        </widget>
        <widget class="QPushButton" name="pushButton_3">
         <property name="geometry">
          <rect>
           <x>630</x>
           <y>30</y>
           <width>250</width>
           <height>250</height>
          </rect>
         </property>
         <property name="minimumSize">
          <size>
           <width>250</width>
           <height>250</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>25</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton {
    color: #333;
    border: 0px solid #555;
    border-radius: 125px;
    border-style: outset;
    background: white;
    padding: 5px;
    }
</string>
         </property>
         <property name="text">
          <string>32oz
946ml</string>
         </property>
         <attribute name="buttonGroup">
          <string notr="true">buttonGroup</string>
         </attribute>
        </widget>
       </widget>
       <zorder>productImage</zorder>
       <zorder>back</zorder>
       <zorder>label_2</zorder>
       <zorder>buttonGroup_2</zorder>
      </widget>
      <widget class="QWidget" name="dispense">
       <property name="minimumSize">
        <size>
         <width>1900</width>
         <height>1080</height>
        </size>
       </property>
       <widget class="QProgressBar" name="progressBar">
        <property name="geometry">
         <rect>
          <x>670</x>
          <y>320</y>
          <width>931</width>
          <height>201</height>
         </rect>
        </property>
        <property name="layoutDirection">
         <enum>Qt::LeftToRight</enum>
        </property>
        <property name="maximum">
         <number>100</number>
        </property>
        <property name="value">
         <number>100</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="format">
         <string> %v ml</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="thanks">
       <property name="minimumSize">
        <size>
         <width>1900</width>
         <height>1080</height>
        </size>
       </property>
       <widget class="QLabel" name="thankstext">
        <property name="geometry">
         <rect>
          <x>600</x>
          <y>340</y>
          <width>591</width>
          <height>101</height>
         </rect>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="font">
         <font>
          <pointsize>30</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Thank you for using Drinkfill!</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="error">
       <property name="minimumSize">
        <size>
         <width>1900</width>
         <height>1080</height>
        </size>
       </property>
       <widget class="QTextEdit" name="textEdit">
        <property name="geometry">
         <rect>
          <x>120</x>
          <y>160</y>
          <width>621</width>
          <height>231</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>20</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">background: transparent;
</string>
        </property>
        <property name="html">
         <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:20pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p align=&quot;center&quot; style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'Ubuntu'; font-size:24pt;&quot;&gt;There has been an error. Sorry for the inconvenience, please wait for someone to come service this machine.&lt;/span&gt;&lt;/p&gt;
&lt;p align=&quot;center&quot; style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-family:'Ubuntu'; font-size:24pt;&quot;&gt;&lt;br /&gt;&lt;/p&gt;
&lt;p align=&quot;center&quot; style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'Ubuntu'; font-size:24pt;&quot;&gt;Thank you for your patience!&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>coffee</tabstop>
  <tabstop>kombucha</tabstop>
  <tabstop>textEdit</tabstop>
  <tabstop>back</tabstop>
 </tabstops>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="buttonGroup"/>
 </buttongroups>
</ui>
